project.clj

```diff
@@ -145,9 +146,13 @@
                     :resource-paths ^:replace ["resources" "libs/datsync/resources"]}]
              :prod {:cljsbuild
                     {:builds
-                     {:client {:source-paths ^:replace ["src/dat/sys/client"]
-                               :compiler {;;:main dat.sys.client.start
-                                          ;;:source-map false
+                     {:client {:source-paths ^:replace ["src/dat/sys/client"] ; no "dev", so no figwheel
```

Comment:
Sorry -- it wasn't completely obvious to me what this was doing at first read (using a source path not including `dev`, which is where all the figwheel-dependent bits live). That said, if you think it's a bit too obvious for folks who are already reading context, I'm happy to pull that comment back out.